src/h3/memview_int.pyx

```diff
@@ -74,8 +76,8 @@ cpdef H3int parent(H3int h, res=None) except 1:
 cpdef int distance(H3int h1, H3int h2) except -1:
     """ compute the hex-distance between two hexagons
     """
-    check_addr(h1)
-    check_addr(h2)
+    check_cell(h1)
+    check_cell(h2)
```

Comment:
nit: will this produce an error that mentions the argument name? Might want to mention either the first or second index are invalid

Reply:
That's a good point. It currently doesn't. Even worse, the error is reported in decimal instead of the hexadecimal string.

I don't see a quick fix off the top of my head, so I'll move this to an issue to think about more deeply.

Reply:
Added: https://github.com/uber/h3-py/issues/78